src/writer/settings.ts

```diff
@@ -80,6 +81,26 @@ export async function verify(body): Promise<any> {
     return Promise.reject(`max number of strategies is ${strategiesLimit}`);
   }
 
+  try {
+    const strategiesList = await (await fetchWithKeepAlive(`${scoreAPIUrl}/api/strategies`)).json();
+
+    msg.payload.strategies
+      .map(strategy => strategy.name)
```

Comment:
there is also one more level of strategies,for example using `multichain` inside `delegation` 🙈